src/integration/common/paragraph.js

```diff
@@ -0,0 +1,12 @@
+export default () => {
+  describe('Paragraph', () => {
+    const paragraphEl = document.querySelector('p');
```

Comment:
Let's check for the presence of a `p` within `main`

```suggestion
    const paragraphEl = document.querySelector('main p');
```

Reply:
I have needed to get the second p to avoid timestamps and copyright text